CTFd/forms/auth.py

```diff
@@ -0,0 +1,32 @@
+from wtforms import PasswordField, StringField, SubmitField
+from wtforms.fields.html5 import EmailField
+from wtforms.validators import InputRequired
+
+from CTFd.forms import BaseForm
+
+
+class RegistrationForm(BaseForm):
+    name = StringField("User Name", validators=[InputRequired()])
```

Comment:
These weren't required before but it makes sense to keep them as required. If you want to modify the behavior of a form you should make another form.